style_editor.html

```diff
@@ -9475,19 +9478,30 @@
 
   // Set up the event listener for the menu
   menu.addEventListener('change', function() {
-    // Get the selected option value
-    const value = menu.value;
-
     // If the selected value is not the default, enable the button and set its action
-    if (value !== '') {
+    if (menu.value !== '') {
       button.disabled = false;
-      button.onclick = actions[value];
+      button.onclick = function() {
+        AddClickedEffect(menu.value);
```

Comment:
AddClickedEffect doesn't take a value.

Reply:
I think this should be sufficient:

button.onclick = AddClickedEffect;

If that doesn't work, do this:

button.onclick = function() {
  AddClickedEffect();
};

Reply:
I swear I tried that first. ok done.
We can also lose the actions[] object then.

Reply:
> We can also lose the actions[] object then.

Indeed, but it doesn't look like you did?

Reply:
NM, I just missed it.